packages/jsx/src/index.test.tsx

```diff
@@ -303,7 +303,7 @@ it('custom component', setup((ctx, h, hf, mount, parent) => {
   const Component = (props: JSX.HTMLAttributes) => <div {...props} />
 
   assert.instance(<Component />, window.HTMLElement)
-  assert.is(((<Component draggable />) as HTMLElement).draggable, true)
+  assert.is(((<Component draggable="true" />) as HTMLElement).draggable, true)
```

Comment:
Attribute is not boolean
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/draggable